opendbc/car/ford/radar_interface.py

```diff
@@ -229,6 +245,79 @@ def _update_delphi_mrr(self, ret: structs.RadarData):
     if headerScanIndex != 3:
       return False
 
+    return self.do_clustering()
+
+  def _update_delphi_mrr_64(self, ret: structs.RadarData):
+    # There is not discovered MRR_Header_InformationDetections message in CANFD
+    # headerScanIndex = int(self.rcp.vl["MRR_Header_InformationDetections"]['CAN_SCAN_INDEX']) & 0b11
+    headerScanIndex = int(self.rcp.vl["MRR_Detection_001"]['CAN_SCAN_INDEX_2LSB_01_01'])
+
+    # In reverse, the radar continually sends the last messages. Mark this as invalid
+    if (self.prev_headerScanIndex + 1) % 4 != headerScanIndex:
+      self.radar_unavailable_cnt += 1
+    else:
+      self.radar_unavailable_cnt = 0
+    self.prev_headerScanIndex = headerScanIndex
```

Comment:
are you copying this from CAN or is this true?

Reply:
20s? You mean 2s? If you right click and show points, you can see that the data is missing, not that's it's static... It's from the fingerprinting process multiplexing to the OBD port (if plugged in).

Fine to remove this reverse code since it's not an issue